ui/partials/perf/graphsctrl.html

```diff
@@ -54,6 +54,7 @@
   <div id="graph-tooltip" ng-class="{locked: selectedDataPoint}">
     <div class="body">
       <div>
+        <span id='close-popup'>X</span>
```

Comment:
I don't think we need to create a whole new button. How about just a new css class that you add to the element? Something like:

```css
graphchooser-close {
    color: #fff;
}
```